### PR TITLE
Fix typos to unify the style

### PR DESCRIPTION
**Pull request by bmcelvee**
Manual CP from https://github.com/openshift/openshift-docs/pull/38475.

### CODE REVIEW

File: modules/dedicated-cluster-install-deploy.adoc [hunk 1]
```diff
@@ -73,8 +73,9 @@ the *Status* column for any errors or failures.
 
 .. Click the installed *Red Hat OpenShift Logging* Operator.
 
-.. Under the *Overview* tab, click *Create Instance* . Paste the following YAML
```
Comment (bmcelvee) [inline modules/dedicated-cluster-install-deploy.adoc:76]: Need verification that this is correct for 4.10. It appears to have been changed in: https://github.com/openshift/openshift-docs/pull/26271/files

File: modules/dedicated-managing-dedicated-administrators.adoc [hunk 2]
```diff
@@ -11,10 +11,10 @@ link:https://console.redhat.com/openshift[{cloud-redhat-com}] site.
 
 [id="dedicated-administrators-adding-user_{context}"]
 == Adding a user
-. Navigate to the *Cluster Details* page and *Users* tab.
```
Comment (bmcelvee) [inline modules/dedicated-managing-dedicated-administrators.adoc:14]: Need to verify that this is correct for 4.10. In appears to have been changed in: https://github.com/openshift/openshift-docs/pull/27872/files.

File: modules/dedicated-cluster-install-deploy.adoc [hunk 3]
```diff
@@ -73,8 +73,9 @@ the *Status* column for any errors or failures.
...
-definition into the window that displays.
+.. Under the *Details* tab, in the *Provided APIs* section, in the
+*Cluster Logging* box, click *Create Instance* . Select *YAML View* and paste the following YAML definition into the window
```
Comment (sagidlow) [inline modules/dedicated-cluster-install-deploy.adoc:77]: The "Select YAML View and paste the following YAML definition into the window..." should be on its own line as a separate step.

**Review by sagidlow — COMMENTED** [review]
One tiny fix needed, otherwise LGTM!

File: modules/dedicated-cluster-install-deploy.adoc (hunk 1 above)
Comment (bmcelvee) [reply modules/dedicated-cluster-install-deploy.adoc:76]: Leaving comments in place. These files appear to be unused, but I'm going to continue with the CP for consistency with main.

File: modules/dedicated-managing-dedicated-administrators.adoc (hunk 2 above)
Comment (bmcelvee) [reply modules/dedicated-managing-dedicated-administrators.adoc:14]: Leaving comments in place. These files appear to be unused, but I'm going to continue with the CP for consistency with main.

File: modules/dedicated-cluster-install-deploy.adoc (hunk 3 above)
Comment (bmcelvee) [reply modules/dedicated-cluster-install-deploy.adoc]: Fixed and added missing `.`. Thanks!

**Pull request merged**